py/desispec/io/frame.py

```diff
@@ -91,6 +91,19 @@ def write_frame(outfile, frame, header=None, fibermap=None, units=None):
 
     return outfile
 
+def read_meta_frame(filename):
+    """ 
+    Args:
+        filename: path to a file, or (night, expid, camera) tuple where
```

Comment:
Is this function really needed as a future abstraction layer, or could you just call `astropy.io.fits.getheader(filename)` for now?

If you do keep it, at minimum you should also include an optional `extname` and correct the docstring (the code doesn't accept a (night,expid,camera) tuple but the docstring says it does).